api/src/database.js

```diff
@@ -5,8 +5,8 @@ const db = knex({
   connection: {
     host: process.env.PGHOST || "localhost",
     user: process.env.PGUSER || "postgres",
-    password: process.env.PGPASSWORD || "daraz12345",
-    database: process.env.PGDATABASE || "postgres",
```

Comment:
This is not something we should commit. All the changes for password / database locally should be done though env variables. It doesn't make sense to change it in every PR